tests/foreman/cli/test_import.py

```diff
@@ -0,0 +1,45 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Host Collection CLI"""
+
+from ddt import ddt
+from robottelo.test import CLITestCase
+from robottelo.cli.imprt import Import as ImportObj
```

Comment:
@elyezer : I was following [test_user.py](https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/cli/test_user.py) format. Thought it is a convention.
Will drop the renaming